app/services/publishers/vacancies/vacancy_step_process.rb

```diff
@@ -54,4 +54,12 @@ def job_details_steps
       %i[job_details]
     end
   end
+
+  def applying_for_the_job_steps
```

Comment:
Is it possible to reuse this logic in `override_enable_job_applications` so that we don't have to remember to update it in both places?

Reply:
the form model does not know about the vacancy_step_process service